lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -206,8 +211,25 @@ def delete_channel_subscriptions(channel_or_id)
         redis.del(redis_key(CHANNEL_PREFIX) + channel_id)
       end
 
+      def trigger(event_name, args, object, **elements)
+        unless config.use_async_broadcasting
+          return trigger_sync(event_name, args, object, **elements)
+        end
+
+        executor_class_job.perform_later(
+          serialized_arguments,
```

Comment:
We should not serialize arguments; it's the responsibility of Active Job; we may assume that arguments are GlobalID identifiable, so we can pass them as is.

Reply:
yea, currently I do not serialize them, but I still collected them (arguments)